pkg/util/coverage/coverage.go

```diff
@@ -0,0 +1,85 @@
+// +build coverage
```

Comment:
maybe add a `pkg/util/coverage/OWNERS` file too?

Reply:
Containing who?

(as not-a-member, it presumably can't be me.)

Reply:
Why don't we fix you not being a member, file an issue on k/org